app/assets/stylesheets/test_helper.scss

```diff
@@ -1,9 +0,0 @@
-@import "vendor/qunit";
-
-.modal-backdrop {
-  display: none;
-}
-
-#reply-control {
-  transition-property: none !important;
-}
```

Comment:
I guess this stuff isn't needed any more?

Reply:
I guess so! It used to be loaded in the main QUnit, but that isn't the case anymore, and nobody complained so we probably don't need it in theme component tests 😃